src/main/webapp/script.js

```diff
@@ -11,3 +11,88 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/*
+ * Displays navbar authentication buttons according to login status.
+ */
+function fetchLogin() {
+  fetch('/login').then(response => response.json()).then(user => {
+    // If user is logged in, show logout button in navbar.
+    if (user.loggedIn) {
+      // If logged in user has no nickname, redirect to nickname setup page.
+      if(!user.isUserRegistered) {
```

Comment:
same as in the other files, place comments inside of the if bracket
```suggestion
    if (user.loggedIn) {
      // If user is logged in, show logout button in navbar.
      if(!user.isUserRegistered) {
        // If logged in user has no nickname, redirect to nickname setup page.
```

Reply:
> But these comments describe the whole if block, so I think they should precede the if statement. Otherwise, it seems I am describing the variables below, don't you think?

Reply:
Done!